config/sync/migrate_plus.migration.soe_stanford_pages_d7_paragraph_rows.yml

```diff
@@ -0,0 +1,60 @@
+uuid: a1c6947a-cb49-4e0e-ad5b-718de2da7210
+langcode: en
+status: true
+dependencies: {  }
+id: soe_stanford_pages_d7_paragraph_rows
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: {  }
+migration_group: stanford_pages_d7
+label: 'SOE Stanford Pages Paragraphs Rows'
+source:
+  constants:
+    text_format: stanford_html
+  plugin: url
+  data_fetcher_plugin: http
+  data_parser_plugin: simple_xml
+  urls:
+    - 'https://engineering.stanford.edu/export/stanford-pages'
+  ids:
+    nid:
+      type: string
+  item_selector: /nodes/node
+  fields:
+  fields:
+    -
+      name: nid
+      label: NID
+      selector: Nid
+    -
+      name: title
+      label: Title
+      selector: Title
+    -
+      name: body
+      label: Body
+      selector: Body
+process:
+  title: title
+  status: constants/status
+  type: constants/type
```

Comment:
Don't need these lines. and `constants/type` is not defined so this would break anyways.